roles/uchiwa/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: install uchiwa
+  yum:
+   name: '{{ uchiwa_package_name }}'
+   state: installed
+  when: manage_packages|default(false)
+
+- name: configure uchiwa
+  template:
+   src: templates/uchiwa.json.j2
+   dest: '{{ uchiwa_file_path }}'
+  notify:
+     - restart uchiwa
+
+- name: ensure uchiwa is started and enabled at boot
+  service:
+   name: '{{ uchiwa_service_name }}'
+   state: started
+   enabled: yes
+  when: manage_services|default(false)
+
+- name: register uchiwa firewal ports
+  set_fact:
+    firewall_ports: >
+      {{ firewall_ports + ['{{ uchiwa_port }}'] }}
```

Comment:
This is incorrect:

    '{{ uchiwa_port }}']

You're already inside a jinja `{{...}}` context.  You just need variable names:

    {{ firewall_ports + [uchiwa_port] }}